source/languages.ttl

```diff
@@ -27,7 +27,8 @@
     rdfs:comment "-1500. \"9fs\" Librisdefinierad MARC-kod."@sv ;
     skos:notation "9fs",
         "9fs"^^kbv:LibrisLocalLanguageCode ;
-    skos:prefLabel "Fornsvenska"@sv .
+    skos:prefLabel "Fornsvenska"@sv ;
+    skos:hiddenLabel "svenska (fornsvenska)" .
```

Comment:
Add language tags on (all) the hidden ones too?

Reply:
I think we could do it for the ones where it is obvious/unambigous.
Need to add `hiddenLabelByLang` to context though.

Reply:
Added that and some obvious tagged hidden values.